python-lib/dku_deeplearning_image/constants.py

```diff
@@ -28,3 +28,6 @@
 DEFAULT_PRED_MIN_THRESHOLD = 0
 COMPILE_LOSS_FUNCTION = 'categorical_crossentropy'
 COMPILE_METRICS = ["accuracy"]
+
+GPU_MEMORY_LIMIT = "memory_limit"
```

Comment:
I would advise doing an Enum or class constants to denote that these are two possible modes of the same parameter

Reply:
Good idea, will do that in another PR to refactor all the constants needing an Enum